zerver/openapi/zulip.yaml

```diff
@@ -18147,13 +18147,7 @@ components:
         The event's type, relevant both for client-side dispatch and server-side
         filtering by event type in [POST /register](/api/register-queue).
     Anchor:
-      oneOf:
-        - type: string
-          enum:
-            - newest
-            - oldest
-            - first_unread
-        - type: integer
+      type: string
```

Comment:
This seems fine; one can imagine display code that does something nice with this sort of type (showing the enumerated values), and it might make openapi-generator produce better bindings, but I think we don't do that presently.

I'm not sure what problem this is solving, though.

Reply:
[Previous discussion](https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/OpenAPI.20requestBody/near/1672619). Added an explanation to the commit message.